### DISCUSSION TITLE
[Esame 23 Gennaio 2019] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#224] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/f4cbaa03-e534-4a0c-80e9-da7d5f022876)

- Tratto da #176

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/971d4148-0e21-4933-95e9-c43997df9373)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/509288f8-f8a0-4380-afd5-ac3434bdb15c)

**Comment by LeoCosta02** [comment]
uguale al foglio6 esercizio 5 . Il calcolo nel punto 1 l'ho fatto con la calcolatrice perchè non saprei svolgerlo a mano
